curiouslearning/FeedTheMonsterJS
# Feat: revamp feed the monster design (FM-272)

# Changes
- fixed issue in the title screen when offline mode
- fix the FTM title in smaller screen device
- fix overlap in FTM title and dev button

# How to test
- Make sure that the title screen show properly in offline mode
- Make sure FTM title fits in one line 
- make sure deb button and title screen don't overlap to each other.

Ref: [FM-272](https://curiouslearning.atlassian.net/browse/FM-272)


[FM-272]: https://curiouslearning.atlassian.net/browse/FM-272?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
